packages/passport-interface/src/Terms.ts

```diff
@@ -0,0 +1,3 @@
+export const LATEST_TERMS = 1;
```

Comment:
maybe have a `MIN_TERMS = 0` in this file just so we're clear on the starting point.